libs/agora/api-description/src/components/schemas/Teams.yaml

```diff
@@ -0,0 +1,7 @@
+type: object
+description: Teams
+properties:
+  teams:
+    type: array
+    items:
+      $ref: Team.yaml
```

Comment:
Seems strange to have to create a yaml definition for an array but top-level arrays are considered bad practice and will trigger a linting error.

Reply:
Returning an object provides more information than an array. In this case, it's clear from the name of the property ("teams") that the value is an array of teams/Teams.